pyccl/tracers.py

```diff
@@ -1,9 +1,68 @@
 from . import ccllib as lib
 from .core import check
-from .background import comoving_radial_distance, growth_rate, growth_factor
+from .background import comoving_radial_distance, growth_rate, growth_factor, Sig_MG
 from .pyutils import _check_array_params, NoneArr
 import numpy as np
 
+def get_MG_transfer_function(cosmo, z, k_MG=None):
+    """ This function allows to obtain the function Sigma(z,k) (1 or 2 D vector)
```

Comment:
Just to avoid confusion, I would say "1 or 2D array" because I think "vector" is commonly taken to be 1D only.

Reply:
Changed!